modules/test/playwright/tests/portal-workflow-task-web/assignments.spec.ts

```diff
@@ -151,3 +161,149 @@ test('send user back to my workflow tasks page after assign another user to revi
 
 	await expect(workflowTasksPage.assignedToMyRolesLink).toBeVisible();
 });
+
+test('logged user must be able to see workflow task at least from a read-only perspective', async ({
+	apiHelpers,
+	configurationTabPage,
+	diagramViewPage,
+	messageBoardsEditThreadPage,
+	messageBoardsPage,
+	messageBoardsWidgetPage,
+	page,
+	processBuilderPage,
+	site,
+	userPersonalBarPage,
+	workflowTaskDetailsPage,
+	workflowTasksPage,
+}) => {
+	const user =
+		await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+			'demo.unprivileged@liferay.com'
+		);
+
+	const defaultUser =
+		await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+			'test@liferay.com'
+		);
+
+	const content = JSON.parse(
+		readFileSync(
+			path.join(
+				__dirname,
+				'../message-boards-web/dependencies/message-board-permissions.json'
+			),
+			'utf-8'
+		)
+	);
+
+	const role = await apiHelpers.headlessAdminUser.postRole({
+		name: 'AdminWorkflowTask' + getRandomInt(),
+		rolePermissions: content,
+		roleType: 'regular',
+	});
+
+	const roleName = role.name;
+
+	await apiHelpers.headlessAdminUser.assignUserToRole(roleName, user.id);
+
+	await messageBoardsWidgetPage.addMessageBoardsPortlet(site);
+
+	await messageBoardsPage.setRoleCategoryPermissions(
+		roleName.toLowerCase(),
+		site.friendlyUrlPath
+	);
+
+	workflowDefinitionName = 'WorkflowDefinition' + getRandomInt();
+	workflowXMLDefinition = readFileSync(
+		__dirname +
+			'/dependencies/administrator-role-assignments-workflow-definition.xml',
+		'utf-8'
+	);
+
+	await apiHelpers.headlessAdminWorkflow.postWorkflowDefinitionSave(
+		workflowDefinitionName,
+		{content: workflowXMLDefinition}
+	);
+
+	await processBuilderPage.goto();
+
+	await processBuilderPage.clickWorkflowDefinitionName(
+		workflowDefinitionName
+	);
+
+	await diagramViewPage.publishWorkflowDefinition();
+
+	await diagramViewPage.goBack();
```

Comment:
Hey @rijema, could you check if this `diagramViewPage.goBack();` is nedded? Since the `configurationTabPage.goTo()` already call the `processBuilderPage.goto();`